src/shared/fonts_utils.js

```diff
@@ -413,16 +413,16 @@ function writeToFile(aBytes, aFilePath) {
   }
 
   netscape.security.PrivilegeManager.enablePrivilege('UniversalXPConnect');
-  var Cc = Components.classes,
+  const Cc = Components.classes,
       Ci = Components.interfaces;
```

Comment:
Here, and in a lot of the other PRs, you'll need to fix the indentation of lines such as these (since the alignment is now wrong).